apps/marginfi-v2-trading/src/components/common/Pool/CreatePoolDialog/CreatePoolDialog.tsx

```diff
@@ -27,6 +27,8 @@ type CreatePoolDialogProps = {
   trigger?: React.ReactNode;
 };
 
+export type SUPPORTED_QUOTE_BANKS = "USDC" | "LST";
```

Comment:
Should this be in a more general place? Is this something that we can reuse somewhere else?

Reply:
Maybe also better to have the pubkey instead of tokenSymbol? or a record/map of pubkey/symbol? Makes it easier to compare/get values no?